### PR TITLE
Exchange Icons Fix / Gold Rename

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Fixes exchange icon downloads. 
fixes icon of Celo native asset on home page
adds meta description to exchange icons page

### Other changes

updates velocity and stability papers to reflect native asset rename

Screen Shot 2020-06-22 at 2 53 56 PM

### Tested

tested the download links on dev.celo.org

### Related issues

- Fixes https://github.com//issues/4130
- Fixes https://github.com/celo-org/celo-labs/issues/529
- Fixes https://github.com/celo-org/celo-labs/issues/528

### CODE REVIEW

File: packages/web/public/static/locales/en/brand.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
   },
   "icons": {
     "title": "Icons",
-    "license": "You may copy and redistribute icons as long as you are abiding by the <0>Community Code of Conduct</0>. You may not remix or transform the icons. ",
+    "license": "You may copy and redistribute icons as long as you are abiding by the <0>Community Code of Conduct</0> and <1>Brand Policy</1>. You may not remix or transform the icons. ",
```
Comment (jmrossy) [inline packages/web/public/static/locales/en/brand.json]: Nit: trailing space

File: yarn.lock [hunk 2]
```diff
@@ -232,6 +232,14 @@
   dependencies:
     cross-fetch "3.0.4"
 
+"@ampproject/toolbox-core@^2.4.0-alpha.1", "@ampproject/toolbox-core@^2.5.4":
```
Comment (jmrossy) [inline yarn.lock]: Lots of changes here. I'm guessing you ran `yarn` from web instead of root?

**Review by jmrossy — COMMENTED** [review]
Looks good, just two comments

File: yarn.lock (hunk 2 above)
Comment (aaronmgdr) [reply yarn.lock]: sounds about right

**Review by jmrossy — APPROVED** [review]

**Pull request merged**